Libplanet.Explorer/GraphTypes/TxResultType.cs

```diff
@@ -0,0 +1,28 @@
+using GraphQL.Types;
+
+namespace Libplanet.Explorer.GraphTypes
+{
+    public class TxResultType : ObjectGraphType<TxResult>
+    {
+        public TxResultType()
+        {
+            Field<NonNullGraphType<TxStatusType>>(
+                nameof(TxResult.TxStatus),
+                description: "The transaction status.",
+                resolve: context => context.Source.TxStatus
+            );
+
+            Field<LongGraphType>(
+                nameof(TxResult.BlockIndex),
+                description: "The block index which the target transaction executed.",
+                resolve: context => context.Source.BlockIndex
+            );
+
+            Field<StringGraphType>(
+                nameof(TxResult.BlockHash),
+                description: "The block hash which the target transaction executed.",
+                resolve: context => context.Source.BlockHash
+            );
```

Comment:
In my thought, the `TxResult.ExceptionName` and the `TxResult.ExceptionMetadata` should be also exposed (Or in the next PR).

Reply:
I think hexadecimal strings seems suit. Because there are many bencodex implementations and it can be handled with them in the client.

Reply:
Added https://github.com/planetarium/libplanet/pull/2130/commits/cbc18e8fb9b5657f7bf505d355309f526a816978 to address [it](https://github.com/planetarium/libplanet/pull/2130#discussion_r936131723).